docker/elg_stance.py

```diff
@@ -133,9 +131,12 @@ def process(twt_source):
         tweet_id = reply.json.get("id_str")
         in_reply_to = reply.json.get("in_reply_to_status_id_str")
 
-        #original = tweets_by_id[in_reply_to]
+        if oblivious_mode:
+            classification, scores = classifier.classify(reply.json)
+        else:
+            original = tweets_by_id[in_reply_to]
+            classification, scores = classifier.classify_with_target(reply.json, original.json)
```

Comment:
This is the core of the difference between TO and TA modes, so it's easy to support both in the same codebase.